application/models/Iota.php

```diff
@@ -172,24 +172,28 @@ function getIotaWorked($location_list, $postdata) {
 		}
 
 		if ($postdata['mode'] != 'All') {
-			$sql .= " and (col_mode = '" . $postdata['mode'] . "' or col_submode = '" . $postdata['mode'] . "')";
+			$sql .= " and (col_mode = ? or col_submode = ?)";
+			$binding[] = $postdata['mode'];
```

Comment:
same here. needed twice

```suggestion
			$binding[] = $postdata['mode'];
            $binding[] = $postdata['mode'];
```